partition/roles/metal-core/templates/metal-core-volumes.j2

```diff
@@ -1,6 +1,6 @@
 {% if metal_stack_switch_os_is_sonic %}
 - /etc/sonic/:/etc/sonic
-- /var/run/redis/:/var/run/redis:ro
+- /var/run/redis/:/var/run/redis
```

Comment:
I don't understand the reason of this change.

Reply:
When writing into the database it cannot be r/o anymore?

Reply:
Good point.